integration/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "npm install && mocha test/*",
+    "test:fast": "mocha test/*"
+  },
+  "author": "",
```

Comment:
probably want to remove it and update the description. The version should probably be `1.0.0`

Reply:
Done